view/src/app/components/dashboard/dashboard-locations/dashboard-locations.component.ts

```diff
@@ -19,8 +19,6 @@ import {GraphicDialogComponent} from '../graphic-dialog/graphic-dialog.component
 export class DashboardLocationsComponent implements OnInit {
   @Input() private location: Locations;
   private localDevices: LocalDevice[] = [];
-  private features: FeatureDTO[] = [];
-  private featuresGraphics: FeatureDTO[] = [];
   public minDate: Date = new Date('01/01/2019 00:00 AM');
   public maxDate: Date = new Date('01/01/2021 12:00 AM');
```

Comment:
Small piece of advice
minDate.setYear(new Date().getYear()-2); 
maxDate.setYear(new Date().getYear()+2);
Fix it later. There is no need to fix it now.